documentation/cgroups_in_storm.md

```diff
@@ -0,0 +1,65 @@
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| storm.cgroup.enable                | This config is used to set whether or not cgroups will be used.  Set "true" to enable use of cgroups.  Set "false" to not use cgroups. When this config is set to false, unit tests related to cgroups will be skipped. Default set to "false"                                                                                                                                                                                                                                                                                         |
+| storm.cgroup.hierarchy.dir   | The path to the cgroup hierarchy that storm will use.  Default set to "/cgroup/storm_resources"                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| storm.cgroup.resources       | A list of subsystems that will be regulated by CGroups. Default set to cpu and memory.  Currently only cpu and memory are supported                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| storm.supervisor.cgroup.rootdir     | The root cgroup used by the supervisor.  The path to the cgroup will be <cgroup.storm.hierarchy.dir>/<cgroup.supervisor.rootdir>.  Default set to "storm"                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
<cgroup.storm.hierarchy.dir> should be <storm.cgroup.hierarchy.dir>. I am assuming that supervisor root dir is actullay same as storm.cgroup.hierarchy.dir and not configurable.